src/Lightweight/SqlQuery/MigrationPlan.hpp

```diff
@@ -155,11 +155,18 @@ enum class SqlPrimaryKeyType : uint8_t
     GUID,
 };
 
+struct SqlForeignKeyDefinition
```

Comment:
```suggestion
struct SqlForeignKeyReferenceDefinition
```
I think a proper term should include that we're referring to the reference part of a foreign key here. it's the other end of an FK only :)